docstore/memdocstore/query.go

```diff
@@ -138,6 +138,19 @@ func compare(x1, x2 interface{}) (int, bool) {
 		}
 		return -1, true
 	}
```

Comment:
Can you add a comment here similar to the one above? I don't follow why you added this. What breaks if you don't?